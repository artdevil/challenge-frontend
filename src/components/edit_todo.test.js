import React from 'react';
import { shallow } from 'enzyme';

import EditTodo from './edit_todo';

describe('<EditTodo />', () => {
  it('shallow edit_todo renders without crashing', () => {
    const wrapper = shallow(<EditTodo showModal={false} hideModal={() => {}} showModalObj={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
