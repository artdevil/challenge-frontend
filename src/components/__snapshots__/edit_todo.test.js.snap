// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditTodo /> shallow edit_todo renders without crashing 1`] = `
<SimpleModal
  editTodo={[Function]}
  hideModal={[Function]}
  showModal={false}
  showModalObj={Object {}}
/>
`;
